src/tests/ftest/harness/basic.py

```diff
@@ -27,7 +28,34 @@ def test_always_passes_hw(self):
 
         :avocado: tags=all
         :avocado: tags=hw,large,medium,ib2,small
-        :avocado: tags=harness,harness_basic_test,test_always_passes,test_always_passes_hw
+        :avocado: tags=harness,harness_basic_test,test_always_passes_hw
         :avocado: tags=always_passes
         """
         self.test_always_passes()
+
+    def test_load_mpi(self):
+        """Simple test of apricot test code to load the openmpi module.
+
+        :avocado: tags=all
+        :avocado: tags=harness,harness_basic_test,test_load_mpi
+        :avocado: tags=load_mpi
```

Comment:
No need to changes, but FYI that we've started explicitly adding `vm` to all non-`hw` tests. Though, most `vm` tests are non-conforming currently.

Reply:
Why though?  We had discussed this way back when defining the tags initially and came to the conclusion that `vm` == `-hw` without the error-proneness of having to remember to add `vm` tags explicitly.

Reply:
This was part of a larger discussion about test tag improvements while you were gone. Most of it has been tabled for now, but some points:
- it's problematic for manual tests that don't work in CI because if the test or feature tags are included, CI will try to run it on a VM and it will fail
- We have to remember to add `hw`, `small`, `medium`, `large`, so it makes sense to add `vm` to be consistent
- I have found a few (rare) cases where a test was _**meant**_ to have `hw,large`, but it just had `large`, which ran on a VM. And it worked "most of the time" only because the config was simple
- Not having a `vm` tag makes it difficult to run the same test on BOTH VM and hardware. To run on hardware, a test MUST have a `hw` tag. To run on a VM, a test CANNOT have a `hw` tag.

The general idea is that `vm` and `hw` should be less about "this represents the test config" and more about "this is specifically where I want this test to run in CI"